migrations/0002_update_outlier_column_structure_w_iasworld_2024_update.py

```diff
@@ -0,0 +1,173 @@
+    Transforms old structure with sv_outlier_type
+    to new structure with 3 separate outlier reason columns
+    """
+    # Insert new columns filled with NaN
+    pos = df.columns.get_loc("sv_outlier_type") + 1
+    for i in range(1, 4):
+        df.insert(pos, f"sv_outlier_reason{i}", np.nan)
+        pos += 1
+
+    print(df.dtypes)
+    # Use the dictionary to populate the new columns
+    for key, value in recode_dict.items():
+        mask = df["sv_outlier_type"] == key
+        for col, val in value.items():
+            df.loc[mask, col] = val
+    print(df.dtypes)
+
+    df = df.drop(columns=["sv_outlier_type"])
+
+    return df
+
+
+def write_dfs_to_s3(dfs, bucket, table):
+    """
+    Writes dictionary of dfs to bucket
+    """
+
+    for df_name, df in dfs.items():
+        file_path = f"{bucket}/0002_update_outlier_column_structure_w_iasworld_2024_update/new_prod_data/{table}/{df_name}.parquet"
+        wr.s3.to_parquet(
+            df=df, path=file_path, index=False, dtype={"sv_outlier_reason3": "string"}
+        )
+
+
+dfs_flag = read_parquet_files_from_s3(
+    os.path.join(
+        os.getenv("AWS_S3_WAREHOUSE_BUCKET"),
+        "sale",
+        "flag",
+    )
+)
+
+recode_dict = {
+    "PTAX-203 flag (Low)": {
+        "sv_outlier_reason1": "PTAX-203 Exclusion",
+        "sv_outlier_reason2": np.nan,
+    },
+    "PTAX-203 flag (High)": {
+        "sv_outlier_reason1": "PTAX-203 Exclusion",
+        "sv_outlier_reason2": np.nan,
+    },
+    "Non-person sale (low)": {
+        "sv_outlier_reason1": "Low price",
+        "sv_outlier_reason2": "Non-person sale",
+    },
+    "Non-person sale (high)": {
+        "sv_outlier_reason1": "High price",
+        "sv_outlier_reason2": "Non-person sale",
+    },
+    "High price (raw)": {
+        "sv_outlier_reason1": "High price",
+        "sv_outlier_reason2": np.nan,
+    },
+    "Low price (raw)": {
+        "sv_outlier_reason1": "Low price",
+        "sv_outlier_reason2": np.nan,
+    },
+    "Anomaly (high)": {
+        "sv_outlier_reason1": "High price",
+        "sv_outlier_reason2": "Statistical Anomaly",
+    },
+    "Anomaly (low)": {
+        "sv_outlier_reason1": "Low price",
+        "sv_outlier_reason2": "Statistical Anomaly",
+    },
+    "Low Price (raw & sqft)": {
+        "sv_outlier_reason1": "Low price",
+        "sv_outlier_reason2": "Low price per square foot",
+    },
+    "High price (raw and sqft)": {
+        "sv_outlier_reason1": "High price",
+        "sv_outlier_reason2": "High price per square foot",
+    },
+    "High price (sqft)": {
+        "sv_outlier_reason1": "High price per square foot",
+        "sv_outlier_reason2": np.nan,
+    },
+    "Low price (sqft)": {
+        "sv_outlier_reason1": "Low price per square foot",
+        "sv_outlier_reason2": np.nan,
+    },
+    "Home flip sale (high)": {
+        "sv_outlier_reason1": "High price",
+        "sv_outlier_reason2": "Price swing / Home flip",
+    },
+    "Home flipe sale (low)": {
```

Comment:
**[Nitpick, blocking]** This typo might account for the missing reasons I noted in the `data-architecture` PR:

```suggestion
    "Home flip sale (low)": {
```